src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -30,6 +30,17 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Core::CursorStyle)
     };
 };
 
+JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Core::AdjustTextMode)
+{
+    static constexpr std::array<pair_type, 5> mappings = {
+        pair_type{ "never", ValueType::Never },
+        pair_type{ "indexed", ValueType::Indexed },
+        pair_type{ "always", ValueType::Always },
+        pair_type{ "true", ValueType::Indexed },
+        pair_type{ "false", ValueType::Never }
+    };
+};

```

Comment:
Actually, since we're switching from supporting a bool to an enum, you're gonna have to update this. Check out CloseOnExitMode. Good example:
https://github.com/microsoft/terminal/blob/5926cc29e22c40715a0003aed47aae5f5fab4877/src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h#L109-L133

Reply:
Thanks, fixed!